public/employers/amazon.yml

```diff
@@ -173,3 +173,20 @@ citations:
       - source: The Independent
         link: https://web.archive.org/web/20200501051715/https://www.independent.co.uk/news/world/americas/amazon-earnings-q2-jeff-bezos-shareholders-profit-coronavirus-a9493466.html
         date: 2020-05-01T07:29:48+01:00
+  - summary: Amazon terminated a program allowing employees to take unlimited, unpaid time off if fearful of contracting COVID-19. Employees instead would have to apply for a leave of absence, around which the requirements for such requests were not made clear.
+    positivity: -1
+    type: publication
+    sources:
+      - source: Vice
+        link: https://web.archive.org/web/20200504165251/https://www.vice.com/en_us/article/z3bjpj/amazon-vp-tim-bray-resigns-calls-company-chickenshit-for-firing-protesting-workers
+        date: 2020-05-04T12:09:14Z
+      - source: CBS News
+        link: https://web.archive.org/web/20200503225347/https://www.cbsnews.com/news/amazon-unlimited-unpaid-time-off/
+        date: 2020-04-28T19:01:00-04:00
+  - summary: An Amazon vice president of six years quit his job in "dismay" after claiming that two more employees had been fired for speaking out against working conditions, and that the situation had become "a vein of toxicity running through the company culture."
```

Comment:
I feel like the weight of the importance of this person leaving is somewhat understated. Tim Bray helped created XML and some other web standards. It may be too long to put that in our claims, I just wish news articles would do him justice.